### PR TITLE
Temp chart

### LINKED ISSUE #198: Explore alternative visual for c2p3 matrix
* show only map, w/ temporal slider, coloring reaches by temperature for each month/day?
* change color scale b/c different variable?

### PR DESCRIPTION
**Pull request by hcorson-dosch-usgs**
* This PR replaces the temperature matrix with an interactive timeseries chart built using the precoded svg PR'd in #234 

Note: there is still a bug with the spatial interaction -- for some segments the associated timeseries path doesn't get selected when you hover over that segment, but I'd like to go ahead and get the PR reviewed as is, so that I can work on merging everything together and incorporating Sam's text edits on 4/30. 

Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by hcorson-dosch-usgs** [comment]
Closes #198

File: src/views/set2/Set2.vue [hunk 1]
```diff
@@ -2096,57 +2103,63 @@
...
+          // // based on whether segment has any observations in record
+          // if (self.segmentDict[segment_id].year_count['2019'] == 0) {
+          //     // select the spatial rectangle corresponding to the highlighted segment
+          //     self.d3.selectAll(".c2p3.matrixSpatialRect.seg" + segment_id)
+          //       .style("stroke-width", 0.5)
+          //       .style("opacity", 1)
+          //       .style("stroke", "#e0e0e0")
+          //       .attr("height", 3)
+          //       // raise the spatial rectangle
+          //       .raise()
```
Comment (cnell-usgs) [inline src/views/set2/Set2.vue:2026]: 👀

Comment (cnell-usgs) [reply src/views/set2/Set2.vue:2026]: 😉

File: src/views/set2/Set2.vue [hunk 2]
```diff
@@ -1483,107 +1464,17 @@
...
-          //     .domain(self.myVars_c2p3)
-          //     .padding(0);
-
-          // create linear gradient for lines
-          self.temp_chart_gradient = svgChart.append("linearGradient")
+          // create linear gradient for lines and legend
+          self.temp_chart_gradient = svgChart.append("defs")
+            .append("linearGradient")
             .attr("id", "plasma-vertical")
+            .attr("gradientUnits", "userSpaceOnUse")
```
Comment (cnell-usgs) [inline src/views/set2/Set2.vue:1453]: still amazed it was this simple!

**Review by cnell-usgs — APPROVED** [review]
Looking good. Things are really coming together. Let's not forget to clean up the commented out code before publishing, alright to keep now as we nail down the final interaction. Great job!

File: src/views/set2/Set2.vue (hunk 1 above)
Comment (hcorson-dosch-usgs) [reply src/views/set2/Set2.vue:2026]: 😂 so much commented-out code 😬. I promise I'll clean this all up once we merge together this PR, upstream, and the data 2020 PR!

File: src/views/set2/Set2.vue (hunk 2 above)
Comment (hcorson-dosch-usgs) [reply src/views/set2/Set2.vue:1453]: me too!!

**Pull request merged**